pages/board.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  <h1>게시판 - 1</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>게시판 - 2</h1>
+  </body>
+</html>
```

Comment:
P1 - 꼭 수정해주세요

여기는 게시판-2가 아니라 게시판 -3이 맞는 것 같습니다. 반드시 수정해주세요

```suggestion
<h1> 게시판 -3 </h1>
```

Reply:
다시 생각해보니 게시판 -2가 맞는 것 같아요 :)
LGTM!